### DISCUSSION TITLE
Will sources overwrite older sources?

### DISCUSSION THREAD

**Discussion by beshur** [crowdin/crowdin-cli#448] [Q&A] [answered]
Hello.

Can't find this info anywhere.

If two developers working in different branches add some keys to the crowdin project, will the last developer to upload sources overwrite the new keys of the first one? Or will it be merged somehow?

(using `crowdin upload sources`)

**Comment by ira-sem** [comment, marked as answer]
Hi @beshur ! If someone changes the key of the string, this string will be considered as totally new in Crowdin project. So for example if there is a string:
"key1":"source string"
but later developer change the key to 
"**K**ey1": "source string"
the previous string is deleted and this one with the changed key is added as a new one

**Reply by beshur** [reply]
thank you

**Comment by andrii-bodnar** [comment]
@beshur in the case of branches both developers should upload files with the `-b` parameter

You can read more about versions management in the article - https://support.crowdin.com/versions-management/ or https://support.crowdin.com/cli-tool/#versions-management

**Reply by beshur** [reply]
> @beshur in the case of branches both developers should upload files with the `-b` parameter
> 
> You can read more about versions management in the article - https://support.crowdin.com/versions-management/ or https://support.crowdin.com/cli-tool/#versions-management

Thank you.
It's probably a bit extreme for now for us.
